pkg/kubelet/container_bridge.go

```diff
@@ -82,6 +82,16 @@ func ensureCbr0(wantCIDR *net.IPNet) error {
 		}
 		return createCBR0(wantCIDR)
 	}
+	// Put the container bridge into promiscuous mode to force it to accept hairpin packets.
```

Comment:
i  think one of my comments got buried.. we should set the bridge to promiscuous the first time around as well, since it captures our intent correctly..